src/auto/cclosure_expression.rs

```diff
@@ -0,0 +1,28 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir)
+// from gir-files (https://github.com/gtk-rs/gir-files)
+// DO NOT EDIT
+
+use glib::translate::*;
+use gtk_sys;
+use std::fmt;
+use Expression;
+
+glib_wrapper! {
+    pub struct CClosureExpression(Object<gtk_sys::GtkCClosureExpression, CClosureExpressionClass>) @extends Expression;
```

Comment:
I think this shouldn't be generated. AFAIU this is for wrapping C function pointers? `GtkClosureExpression` would be the one for us here :)